fission/src/ui/panels/configuring/scoring/ScoringZonesPanel.tsx

```diff
@@ -1,19 +1,32 @@
-import { useState } from "react"
+import { useEffect, useState } from "react"
 import { usePanelControlContext } from "@/ui/PanelContext"
 import Button from "@/components/Button"
 import Label, { LabelSize } from "@/components/Label"
 import Panel, { PanelPropsImpl } from "@/components/Panel"
 import ScrollView from "@/components/ScrollView"
 import Stack, { StackDirection } from "@/components/Stack"
-import { ScoringZone } from "./ZoneConfigPanel"
+import { ScoringZonePreferences } from "@/systems/preferences/PreferenceTypes"
+import PreferencesSystem from "@/systems/preferences/PreferencesSystem"
+import SynthesisBrain from "@/systems/simulation/synthesis_brain/SynthesisBrain"
+import { AiOutlinePlus } from "react-icons/ai"
+import { IoPencil, IoTrashBin } from "react-icons/io5"
+
+const AddIcon = <AiOutlinePlus size={"1.25rem"} />
+const DeleteIcon = <IoTrashBin size={"1.25rem"} />
+const EditIcon = <IoPencil size={"1.25rem"} />
 
 type ScoringZoneRowProps = {
-    zone: ScoringZone
+    zone: ScoringZonePreferences
     openPanel: (id: string) => void
     deleteZone: () => void
+    saveZones: () => void
+}
+
+export class SelectedZone {
```

Comment:
Really not a fan of this, but passing in parameters to other panels isn't super doable atm, from what I know. @PepperLola Is it possible?

Reply:
I'm becoming more and more in favor of this global route for UI solutions now...